### ISSUE TITLE
Asses the impact of precision on the ML model prediction

### ISSUE THREAD

**Issue by srikanthallu** [ROVI-org/storage-requirement-estimation#5]
Study the impact of single and mixed precision on the performance predictions of the ML models. Need to understand from the data point of view used for training if single precision suffices ? or do we need to embrace double precision data with mixed precision models.

**Comment by WardLT** [comment]
Related to #1 ?

**Comment by srikanthallu** [comment]
#1 is referring to compression which could be different precision of data, right ?

**Comment by WardLT** [comment]
Yes, though it could include other types of lossy compression (e.g., sampling the data at lower frequencies)
